.github/workflows/release.yaml

```diff
@@ -1,46 +1,47 @@
-name: release
+name: Release KDIS
```

Comment:
Leave as is because the name implies a KDIS release. Furthermore, the name might be used to reference this workflow, thus a simple/short name is suggested.